src/cloudservice/azext_cloudservice/manual/_help.py

```diff
@@ -0,0 +1,188 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['cloud-service role-instance'] = """
```

Comment:
hi @qwordy , this RP has lots of manual codes, could you pls share why it need manual customization? then we can share that which part isn't covered by codegen yet.

Reply:
I've created issues on https://github.com/Azure/autorest.az/issues/created_by/qwordy
![image](https://user-images.githubusercontent.com/5019528/112804980-9e9d9e00-90a7-11eb-90b8-0928087d4735.png)